core/src/ast_builder/select/root.rs

```diff
@@ -117,5 +123,9 @@ mod tests {
             .build();
         let expected = "SELECT * FROM Foo WHERE col1 > col2";
         test(actual, expected);
+
+        let actual = table("Foo").select().group_by("Country, Age").build();
```

Comment:
[Question]
Is there a reason to add `group by` test case, not a `order by` test case?